extension/src/cli/dvc/reader.ts

```diff
@@ -47,6 +48,10 @@ export class DvcReader extends DvcCli {
     this
   )
 
+  public dag(cwd: string) {
+    return this.executeDvcProcess(cwd, Command.DAG, Flag.MD)
```

Comment:
If the `dvc.yaml` format is invalid, an error gets thrown: 

<img width="477" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/efbbcf87-5057-4dfa-b54f-f467e9b03a9c">

Reply:
Thought about this after I signed off last night. It's an easy fix.

Reply:
Fixed, see last demo